open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -1,12 +1,27 @@
 {
```

Comment:
<p>the name of function should follow this rule:</p><blockquote><a href="https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function" target="_blank">https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function</a>Function names should be lowercase, with words separated by underscores as necessary to improve readability. Variable names follow the same convention as function names.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Just name this as string_lower()</p>